README.md

```diff
@@ -3,29 +3,34 @@
 Flutter Riverpod snippets is a way to enhance the way you use Riverpod. It contains a collection of different
 snippets such as `provider`.
 
-![greetingProviderGif](gifs/greetingProvider.gif)
+![familyGif](gifs/greetingProvider.gif)
```

Comment:
This line seems to have changed, could we keep the text to "greetingProviderGif" in case the image isn't loaded?